judgments/views.py

```diff
@@ -65,19 +65,27 @@ def browse(request, court=None, subdivision=None, year=None):
 def detail(request, judgment_uri):
     context = {}
     try:
-        results = api_client.eval_xslt(judgment_uri)
-        xml_results = api_client.get_judgment_xml(judgment_uri)
-        multipart_data = decoder.MultipartDecoder.from_response(results)
-        judgment = multipart_data.parts[0].text
-        model = Judgment.create_from_string(xml_results)
-        context["judgment"] = judgment
-        context["page_title"] = model.metadata_name
-        context["judgment_uri"] = judgment_uri
-        context["pdf_size"] = get_pdf_size(judgment_uri)
-    except MarklogicResourceNotFoundError:
+        is_published = api_client.get_published(judgment_uri)
```

Comment:
I think we're kinda fortunate that the page 500'd, because otherwise we'd be publishing unpublished pages... I don't know if there's any improvements we can make to make that less likely?

Reply:
Yeah I was wondering that!! Definitely worth looking into why we missed that